package/windows/build_vagrant_msi.ps1

```diff
@@ -0,0 +1,373 @@
+<#
+.SYNOPSIS
+Packages a Vagrant installer from a substrate artifact and
+an installed artifact
```

Comment:
Are these powershell scripts inherited from the old build process or is it all new?

Reply:
They are new